ranger/config/commands.py

```diff
@@ -1859,3 +1859,34 @@ def tab(self, tabnum):
             in sorted(self.modes.keys())
             if mode
         )
+
+
+class paste_ext(Command):
+    """
+    :paste_ext
+
+    Like paste but renames conflicting files so that the
+    file extension stays intact.
```

Comment:
Maybe admit here that it's best-effort rather than claim it always works : )

Reply:
OK